symengine/ruby/spec/basic_spec.rb

```diff
@@ -16,56 +16,5 @@
                 end
             end
         end
-
-        describe 'binary operations' do
-            before :each do
-                @a = SymEngine::Basic.new
-                @b = SymEngine::Basic.new
-            end
-            describe '#+' do
-                context 'with another Basic object as argument' do
-                    xit 'returns a Basic object that is result of
-                        self added to the argument' do
```

Comment:
I didn't get the `symbols` part. Could you please elaborate? 
Yes, it does make sense. Testing it once in `Basic` will cover it, but what should I initialise `Basic` object with?